### PR TITLE
[improvement](mtmv, statistics)Collect index row count for MTMV.

### PR DESCRIPTION
**Pull request by Jibing-Li**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/TabletStatMgr.java [hunk 1]
```diff
@@ -96,7 +95,8 @@ protected void runAfterCatalogReady() {
             }
             List<Table> tableList = db.getTables();
             for (Table table : tableList) {
-                if (table.getType() != TableType.OLAP) {
+                // Will process OlapTable and MTMV
+                if (!(table instanceof OlapTable)) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/TabletStatMgr.java:99]: use `table.isManaged()` interface

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**